urllib3/exceptions.py

```diff
@@ -144,6 +144,11 @@ class ResponseError(HTTPError):
     SPECIFIC_ERROR = 'too many {status_code} error responses'
 
 
+class NoIPv6Warning(HTTPWarning):
```

Comment:
Can we move this somewhere under test/? I'd rather not pollute the core urllib3 code with test-related things. :)